lib/ansible/modules/cloud/amazon/s3_bucket.py

```diff
@@ -68,6 +68,14 @@
     description:
       - Whether versioning is enabled or disabled (note that once versioning is enabled, it can only be suspended)
     type: bool
+  encryption:
+    description:
+      - Describes the default server-side encryption to apply to new objects in the bucket. If none is specified, the default encryption will be applied.
```

Comment:
People might misinterpret which none you mean. You might want to note that 'none' as a string is necessary to remove the server-side encryption.